src/components/organisms/MobileNav/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react'
+import Logo from "../../atoms/Logo"
+import LanguageOption from "../../atoms/LanguageOption"
+import BurgerIcon from "../../atoms/Burger"
+import MobileNavMenu from "../../pages/MobileNavMenu"
+import data from "../../../../config/translations/en.json"
+import { StyleMobileNavWrapper, StyleLanguageBurgerWrapper } from "./styles"
+
+const importedProps = {
```

Comment:
At the time, I thought it would be easier to keep track of if I kept them all in a single object, but judging by your question, maybe I don't need to? 😅